root/recording/RecordingLayout.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+import type {Node as ReactNode} from 'react';
+
+import Layout from '../layout';
+import RecordingSidebar from '../layout/components/sidebar/RecordingSidebar';
+import {hyphenateTitle, l} from '../static/scripts/common/i18n';
+import {
+  artistCreditFromArray,
+  reduceArtistCredit,
+} from '../static/scripts/common/immutable-entities';
+
+import RecordingHeader from './RecordingHeader';
+
+type Props = {|
+  +children: ReactNode,
+  +entity: RecordingT,
+  +fullWidth?: boolean,
+  +page: string,
+  +title?: string,
+|};
+
+const RecordingLayout = ({
+  children,
+  entity: recording,
```

Comment:
Just noticed this, used in other PRs too. Is it a good practice? Should we generalize using it?

Reply:
It was a suggestion from @mwiencek when we decided to generalize... something, I forget what exactly :) I think it makes it more clear than using entity everywhere, but dunno :)